### PR TITLE
Clarify answer file vs. sample output files

### PR DESCRIPTION
**Pull request by evouga**
Added a Sample Output section and clarified the purpose of the answer file in various places.

### CODE REVIEW

File: spec/2023-07-draft.md [hunk 1]
```diff
@@ -417,7 +417,8 @@ The table below summarizes the supported test data:
 | Extension                       | Described In         | Summary                                |
 |---------------------------------|----------------------|----------------------------------------|
 | `.in`                           | Input                | Input piped to standard input          |
-| `.ans`                          |                      | Expected (AC) answer                   |
+| `.ans`                          | Output Validator     | Answer file used to validate submission output |
```
Comment (JoelNiemela) [inline spec/2023-07-draft.md:420]: Is something like "Input to the Output Validator" maybe more obvious than "Answer file" (which feels a bit misleading to me)

File: spec/2023-07-draft.md [hunk 2]
```diff
@@ -511,6 +512,13 @@ The format of `testdata.yaml` is as follows:
 | output_validator_flags | String                              | empty string            | Inherited      | Arguments passed to the output validator for this test data group.
 | run_samples            | Boolean                             | true                    | Not applicable | Signifies whether samples should be run, see below for details.
 
+### Sample Output
+
+Test cases in the `sample` test data group may optionally include an `.out` file. This file contains example correct output for the sample input in the corresponding `.in` file. This output is displayed in the problem statement (except if the problem statement is in PDF format---[See Problem Statements](#problem-statements)) and the sample output file must pass output validation given the test case's `.ans` file.
+
+For problems that use a default output validator where the `.ans` file is the output of a trusted judge reference solution, the `.out` file may be ommitted, in which case the contents of the `.ans` file will be used as the sample output.
```
Comment (JoelNiemela) [inline spec/2023-07-draft.md:519]:
```suggestion
The `.out` file may be omitted, in which case the `.ans` file will be used as the `.out` file.
In this case, the `.ans` file must pass output validation.
```

File: spec/2023-07-draft.md (hunk 1 above)
Comment (JoelNiemela) [inline spec/2023-07-draft.md:420]:
```suggestion
| `.ans`                          | Output Validator     | Input to the Output Validator                  |
```

File: spec/2023-07-draft.md [hunk 3]
```diff
@@ -511,6 +512,14 @@ The format of `testdata.yaml` is as follows:
 | output_validator_flags | String                              | empty string            | Inherited      | Arguments passed to the output validator for this test data group.
 | run_samples            | Boolean                             | true                    | Not applicable | Signifies whether samples should be run, see below for details.
 
+### Sample Output
+
+Test cases in the `sample` test data group may optionally include an `.out` file. This file contains example correct output for the sample input in the corresponding `.in` file. This output is displayed in the problem statement (except if the problem statement is in PDF format---[See Problem Statements](#problem-statements)) and the sample output file must pass output validation given the test case's `.ans` file.
```
Comment (JoelNiemela) [inline spec/2023-07-draft.md:517]:
```suggestion
Test cases in the `sample` test data group may optionally include an `.out` file. This file contains example correct output for the sample input in the corresponding `.in` file. This output is displayed in the problem statement (except if the problem statement is in PDF format—[See Problem Statements](#problem-statements)) and the sample output file must pass output validation given the test case's `.ans` file.
```

File: spec/2023-07-draft.md [hunk 4]
```diff
@@ -466,7 +467,7 @@ illustration | image     | `.png`, `.jpg`, `.jpeg`, or `.svg` | privileged infor
 
 - An *illustration* provides a visualization of the associated test case.
   Note that at most one image file may exist for each test case.
-
+
```
Comment (niemela) [inline spec/2023-07-draft.md:470]:
```suggestion

```

File: spec/2023-07-draft.md [hunk 5]
```diff
@@ -511,6 +512,14 @@ The format of `testdata.yaml` is as follows:
 | output_validator_flags | String                              | empty string            | Inherited      | Arguments passed to the output validator for this test data group.
 | run_samples            | Boolean                             | true                    | Not applicable | Signifies whether samples should be run, see below for details.
 
+### Sample Output
+
+Test cases in the `sample` test data group may optionally include an `.out` file. This file contains example correct output for the sample input in the corresponding `.in` file. This output is displayed in the problem statement (except if the problem statement is in PDF format—[See Problem Statements](#problem-statements)) and the sample output file must pass output validation given the test case's `.ans` file.
```
Comment (niemela) [inline spec/2023-07-draft.md:517]:
```suggestion
Test cases in the `sample` test data group may optionally include an `.out` file. 
This file contains example correct output for the sample input in the corresponding `.in` file. 
This output is displayed in the problem statement (except if the problem statement is in PDF format—[See Problem Statements](#problem-statements)) and the sample output file must pass output validation given the test case's `.ans` file.
```

File: spec/2023-07-draft.md [hunk 6]
```diff
@@ -511,6 +512,14 @@ The format of `testdata.yaml` is as follows:
...
+
+The `.out` file may be omitted, in which case the `.ans` file will be used as the `.out` file.
+In this case, the `.ans` file must pass output validation.
+
+
```
Comment (niemela) [inline spec/2023-07-draft.md:522]:
```suggestion
```

**Review by niemela — APPROVED** [review]

File: spec/2023-07-draft.md (hunk 5 above)
Comment (JoelNiemela) [inline spec/2023-07-draft.md:517]:
```suggestion
Test cases in the `sample` test data group may optionally include an `.out` file. This file contains example correct output for the sample input in the corresponding `.in` file. This output is displayed in the problem statement (except if the problem statement is in PDF format — [See Problem Statements](#problem-statements)) and the sample output file must pass output validation given the test case's `.ans` file.
```

File: spec/2023-07-draft.md [hunk 7]
```diff
@@ -511,6 +511,15 @@ The format of `testdata.yaml` is as follows:
 | output_validator_flags | String                              | empty string            | Inherited      | Arguments passed to the output validator for this test data group.
 | run_samples            | Boolean                             | true                    | Not applicable | Signifies whether samples should be run, see below for details.
 
+### Sample Output
+
+Test cases in the `sample` test data group may optionally include an `.out` file. 
+This file contains example correct output for the sample input in the corresponding `.in` file. 
+This output is displayed in the problem statement (except if the problem statement is in PDF format—[See Problem Statements](#problem-statements)) and the sample output file must pass output validation given the test case's `.ans` file.
```
Comment (JoelNiemela) [inline spec/2023-07-draft.md:518]:
```suggestion
This output is displayed in the problem statement (except if the problem statement is in PDF format — [See Problem Statements](#problem-statements)) and the sample output file must pass output validation given the test case's `.ans` file.
```

**Review by JoelNiemela — APPROVED** [review]

**Pull request merged**